File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    day = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    
    daytime = b - 1;
    for i in range(a - 1):
        daytime += month[i]

    return day[daytime % 7]

File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    const month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    const day = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    
    let daytime = b - 1;
    for(let i = 0; i < a - 1; i++){
        daytime += month[i]
    }

    return day[daytime % 7]
}